platform/windows/godot.natvis

```diff
@@ -111,25 +206,136 @@
+
+	<Type Name="RID">
+		<DisplayString>{{{_id}}}</DisplayString>
+		<Expand>
+			<Item Name="id">_id</Item>
+		</Expand>
+	</Type>
+
+	<Type Name="Node" Inheritable="false">
+		<Expand>
+			<Item Name="Object">(Object*)this</Item>
+			<Item Name="class_name">(StringName*)(((char*)this) + sizeof(Object))</Item>
+			<Item Name="data">(Node::Data*)(((char*)this) + sizeof(Object) + sizeof(StringName))</Item>
+		</Expand>
+	</Type>
+
+	<Type Name="Pair&lt;*,*&gt;">
+		<DisplayString>{{key = {first} value = {second}}}</DisplayString>
+		<Expand>
+			<Item Name="first">first</Item>
+			<Item Name="second">second</Item>
+		</Expand>
+	</Type>
+
+	<Type Name="Dictionary">
+		<DisplayString>{{size = {_p-&gt;variant_map.list._data ? _p-&gt;variant_map.list._data-&gt;size_cache : 0}}}</DisplayString>
+		<Expand>
+			<LinkedListItems>
+			  <Size>_p-&gt;variant_map.list._data ? _p-&gt;variant_map.list._data-&gt;size_cache : 0</Size>
+			  <HeadPointer>_p-&gt;variant_map.list._data-&gt;first</HeadPointer>
+			  <NextPointer>next_ptr</NextPointer>
+			  <ValueNode>value</ValueNode>
+			</LinkedListItems>
+		</Expand>
+	</Type>
+
+	<Type Name="PoolVector&lt;*&gt;">
+		<DisplayString>{{size = {alloc ? alloc-&gt;size / sizeof($T1) : 0}}}</DisplayString>
+		<Expand>
+			<ArrayItems>
+				<Size>alloc ? alloc-&gt;size / sizeof($T1) : 0</Size>
+				<ValuePointer>($T1*)alloc-&gt;mem</ValuePointer>
+			</ArrayItems>
+		</Expand>
+	</Type>
+
+	<Type Name="Vector&lt;*&gt;">
+		<Expand>
+			<Item Name="[size]">_cowdata._ptr ? (((const unsigned int *)(_cowdata._ptr))[-1]) : 0</Item>
+			<ArrayItems>
+				<Size>_cowdata._ptr ? (((const unsigned int *)(_cowdata._ptr))[-1]) : 0</Size>
+				<ValuePointer>_cowdata._ptr</ValuePointer>
+			</ArrayItems>
+		</Expand>
+	</Type>
+
+	<Type Name="LocalVector&lt;*&gt;">
+		<Expand>
+			<Item Name="[size]">count</Item>
+			<ArrayItems>
+				<Size>count</Size>
+				<ValuePointer>data</ValuePointer>
+			</ArrayItems>
+		</Expand>
+	</Type>
+
+	<Type Name="List&lt;*&gt;">
+		<Expand>
+			<Item Name="[size]">_data ? (_data->size_cache) : 0</Item>
+			<LinkedListItems>
+				<Size>_data ? (_data->size_cache) : 0</Size>
+				<HeadPointer>_data->first</HeadPointer>
+				<NextPointer>next_ptr</NextPointer>
+				<ValueNode>value</ValueNode>
+			</LinkedListItems>
+		</Expand>
+	</Type>
+
+	<!-- GDNative -->
+
+	<Type Name="godot::Vector3">
+		<DisplayString>{{{coord[0]},{coord[1]},{coord[2]}}}</DisplayString>
+		<Expand>
+			<Item Name="x">coord[0]</Item>
+			<Item Name="y">coord[1]</Item>
+			<Item Name="z">coord[2]</Item>
+		</Expand>
+	</Type>
+
+	<Type Name="godot::Vector2">
+		<AlternativeType Name="godot::Point2"/>
+		<AlternativeType Name="godot::Size2"/>
+		<DisplayString>{{{x},{y}}}</DisplayString>
 		<Expand>
 			<Item Name="x">x</Item>
 			<Item Name="y">y</Item>
-			<Item Name="z">z</Item>
 		</Expand>
 	</Type>
 
-	<Type Name="Quaternion">
-		<DisplayString>Quaternion {{{x},{y},{z},{w}}}</DisplayString>
+	<Type Name="godot::Quat">
```

Comment:
`Quat` was renamed `Quaternion`, I think for GDNative too.

Reply:
I could not find a single rename that was used in master in godot-cpp. A lot of changes are still not reflected neither in `master` branch or in `3.3` one. This is the same reason why there is not new PackedArray types in this natvis, since they are missing from godot-cpp as well.

Reply:
`godot-cpp` is outdated and @vnen is fully reworking it to match the new extensions system, so this might not be relevant in the `master` branch in the near future.

Might be best to keep these only in the `3.x` version for now and keep the `master` NatVis focused on core types. Once the extensions system is finalized and `godot-cpp` is synced we can see how to add visualization routines for what it will be.

Reply:
`godot-cpp` 4.0 API changes PR (https://github.com/godotengine/godot-cpp/pull/524) is not merged (and with the new Native Extension API in development it probably won't be).